src/profile/Profile.tsx

```diff
@@ -17,7 +19,8 @@ import UitpasInfo from './UitpasInfo/UitpasInfo';
 const Profile = () => {
   const [logOutModalVisible, toggleLogOutModalVisible] = useToggle(false);
   const { data: passHolder, isLoading: isPassHolderLoading } = useGetMe();
-  const [isMIANotificationVisible, setIsMIANotificationVisible] = useState(false);
+  const isUitpasInfoClosedInStorage = useMemo(() => storage.getBoolean(StorageKey.IsUitpasInfoClosed), []);
+  const [isUitpasInfoClosed, setIsUitpasInfoClosed] = useState(isUitpasInfoClosedInStorage);
```

Comment:
Why not just use the storage value as default value for `useState` ?

```suggestion
  const [isUitpasInfoClosed, setIsUitpasInfoClosed] = useState(storage.getBoolean(StorageKey.IsUitpasInfoClosed));
```

It will only be executed once either way 🤷🏻‍♀️